SAMtags.tex

```diff
@@ -246,6 +250,39 @@ \subsection{Barcodes}
 The {\tt BC} tag should match the {\tt QT} tag in length. 
 In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and places a hyphen (`{\tt -}') between the barcodes from the same template. 
 
+\item[BZ:Z:\tagvalue{qualities+}] 
+Phred quality of the (uncorrected) unique molecular identifier sequence in the {\tt OX} tag.
+Same encoding as {\sf QUAL}, i.e., Phred score + 33.
+The {\tt OX} tags should match the {\tt BZ} tag in length. 
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with a space (`{\tt \textvisiblespace}') between the different strings.
+
+\item[CB:Z:\tagvalue{str}]
+
+  Cell identifier based on the (possibly corrected) cellular-barcode
+  sequence ({\tt CR}). The sequence may optionally be followed by a
+  dash ('-') and one or more alphanumeric characters to form an
+  identifier. The respective quality scores are (optionally) stored in
```

Comment:
I don't think that there needs to be discussion of quality scores here. it's enough to have it in CR.